pkg/devcontainer/setup/setup.go

```diff
@@ -325,11 +328,12 @@ func runPostCreateCommand(commands []types.LifecycleHook, user, dir string, remo
 			cmd.Env = os.Environ()
 			cmd.Env = append(cmd.Env, remoteEnvArr...)
 			cmd.Stdout = writer
-			cmd.Stderr = writer
+			cmd.Stderr = errwriter
+			log.Debugf("Executing command %s: %s...", k, cmd.Args)
 			err := cmd.Run()
 			if err != nil {
-				log.Errorf("Failed running command %s: %v", k, err)
-				return err
+				log.Debugf("Failed running postcreate command %s: %v", cmd.Args, err)
```

Comment:
`postcreate` might be a little confusing here. I know we call the function that way but the spec calls these ["lifecycle script"](https://containers.dev/implementors/json_reference/#lifecycle-scripts)